happytransformer/__init__.py

```diff
@@ -3,5 +3,9 @@
 from happytransformer.happy_text_classification import HappyTextClassification
 from happytransformer.happy_next_sentence import HappyNextSentence
 from happytransformer.happy_token_classification import HappyTokenClassification
+from happytransformer.happy_generation import HappyGeneration
+
+from happytransformer.happy_generation import HappyGeneration, GEN_DEFAULT_SETTINGS, GEN_GREEDY_SETTINGS, \
+    GEN_BEAM_SETTINGS, GEN_TOP_K_SAMPLING_SETTINGS, GEN_GENERIC_SAMPLING_SETTINGS
```

Comment:
backslash should be replaced with parentheses

Reply:
2431985